lms/djangoapps/teams/tests/test_views.py

```diff
@@ -296,6 +333,10 @@ def test_page(self):
         self.assertIsNone(result['next'])
         self.assertIsNotNone(result['previous'])
 
+    def test_expand_user(self):
+        result = self.get_teams_list(200, {'expand': 'user', 'topic_id': 'renewable'})
+        self.assertIn('email', result['results'][0]['membership'][0]['user'])
```

Comment:
It uses the same serializer as the User API. id, url, email, name, username, and preferences are returned.